Aptoide/aptoide-client-v8
# [feature/ASV-2119] ABTest apkfy

**What does this PR do?**

   This PR creates an AB Test where we have two groups: one where apkfy users will not see an interstitial on the first install, and one as normal (they will see an ad regardless). It also sends an event to Rakam and sets super properties to measure retention.

**Database changed?**

   No

**How should this be manually tested?**

  First, you have to build the code and insert apkfy related files to test it. So either test in dev version with v1 signing (+apkfy files) or test with release (+apkfy files), if you can see release requests in Charles. Next, you have to test both groups ('control' and 'no_ads_on_first_session') and check for 'control' that interstitial appears on the first install, and that it doesn't for 'no_ads_on_first_session'. Independently of this, check if an event was sent to Rakam analytics with the result of the AB Test. Also check if the current AB Test group is also set on Rakam super properties (and if you force stop aptoide, they should still be there).

  For testing both groups, you can use the rewrite on the JIRA ticket below.

**What are the relevant tickets?**

  [ASV-2119](https://aptoide.atlassian.net/browse/ASV-2119)

**Code Review Checklist**
